File: app/src/main/java/com/google/android/leanbacklauncher/RoundedRectOutlineProvider.java
package com.google.android.leanbacklauncher;

import android.graphics.Outline;
import android.view.View;
import android.view.ViewOutlineProvider;

public class RoundedRectOutlineProvider extends ViewOutlineProvider {
    private final float mRadius;

    public RoundedRectOutlineProvider(float radius) {
        this.mRadius = radius;
    }

    public void getOutline(View view, Outline outline) {
        outline.setRoundRect(0, 0, view.getWidth(), view.getHeight(), this.mRadius);
    }
}

File: app/src/main/java/com/amazon/tv/leanbacklauncher/RoundedRectOutlineProvider.kt
package com.amazon.tv.leanbacklauncher

import android.graphics.Outline
import android.view.View
import android.view.ViewOutlineProvider

class RoundedRectOutlineProvider(private val mRadius: Float) : ViewOutlineProvider() {

    override fun getOutline(view: View, outline: Outline) {
        outline.setRoundRect(0, 0, view.width, view.height, mRadius)
    }

}